spec/requests/api/v2/allocations_spec.rb

```diff
@@ -58,6 +58,29 @@
     end
   end
 
+  describe "DELETE" do
+    context "with an existing allocation" do
```

Comment:
feels wide, is there a reason not restricting or tests covering
- [ ] user association
- [ ] recruitment cycle

Reply:
For the delete method we haven't put any scoping on recruitment cycle - it only locates the record by id. 

For the user association, I've added a test where the user isn't from that accredited body!

Reply:
can add the scope for recruitment cycle then please... ther eis no reason to allow for deletion fo something out of bounds